.github/actions/pipenv/action.yml

```diff
@@ -0,0 +1,12 @@
+name: pipenv
+description: 'Install python packages with pipenv'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install pipenv dependency
+      shell: bash
+      run: pip install -r ${{ github.action_path }}/requirements.txt
+    - name: Prepare pipenv virtualenv
+      shell: bash
+      run: pipenv install --deploy --dev --python $(cat .python-version)
```

Comment:
remember to always add a readme entry for each new action

document also that the .python-version file in the repo root is a requirement (for now)